lib/elixir/lib/io/ansi.ex

```diff
@@ -170,6 +170,30 @@ defmodule IO.ANSI do
   @doc "Sends cursor home."
   defsequence(:home, "", "H")
 
+  @doc "Sends cursor to the absolute position specified by `lines`, `columns`, with 0,0 being the top left corner."
```

Comment:
I would split this line:

```elixir
@doc """
Sends cursor to the absolute position specified by `lines`, `columns`, 
with 0,0 being the top left corner.
"""
```

Reply:
Btw, it seems the formatter leaves that `@doc` tag as is. Should the formatter try to convert `@doc "..."` into `@doc """ (new line) ... """`? Or at least emit a message?